### PR TITLE
php71-apcu-bc 1.0.3 (new formula)

### PR DESCRIPTION
**Pull request by romainneutron**
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-php/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --strict --online <formula>` (after doing `brew install <formula>`)?

-----

This adds apcu-bc for PHP 7.1 (missing at the moment)

### CODE REVIEW

**Comment by romainneutron** [comment]
Any thought on this @javian @ablyler ?

**Comment by javian** [comment]
@romainneutron I sent a review request for a change since the latest version is not used - did you not get that ?

**Comment by romainneutron** [comment]
I'm sorry I don't see any comment or review request, or I do not understand what you mean...
image

**Comment by javian** [comment]
@romainneutron Sorry, I think I might have misunderstood how the review worked so I never committed my proposed changes. They are there now.

**Comment by javian** [comment]
@romainneutron The commit message does not match the content of the PR now. Please submit the update for 7.0 in a different PR and use the notation `php70-apcu-bc <version updated to>`  . Can you also please rename this PR to `php71-apcu-bc 1.0.3 (new formula)` ?

File: Formula/php70-apcu-bc.rb [hunk 1]
```diff
@@ -4,17 +4,10 @@ class Php70ApcuBc < AbstractPhp70Extension
   init
   desc "APC User Cache - BC"
   homepage "https://pecl.php.net/package/apcu_bc"
-  url "https://pecl.php.net/get/apcu_bc-1.0.1.tgz"
-  sha256 "512674d891104d6da91811dbb89d28ab3faa356ee0ab4cbeae9bba9cf3e971cb"
+  url "https://pecl.php.net/get/apcu_bc-1.0.3.tgz"
+  sha256 "40b63ada315ffce81e2e8d75162606090e1cc72fe94207bc7daa6dd260694919"
   head "https://github.com/krakjoe/apcu-bc.git"
 
-  bottle do
```
Comment (javian) [inline Formula/php70-apcu-bc.rb]: You do not have to remove the bottle block - it will be updated automatically.

File: Formula/php70-apcu-bc.rb [hunk 2]
```diff
@@ -4,17 +4,10 @@ class Php70ApcuBc < AbstractPhp70Extension
   init
   desc "APC User Cache - BC"
   homepage "https://pecl.php.net/package/apcu_bc"
-  url "https://pecl.php.net/get/apcu_bc-1.0.1.tgz"
-  sha256 "512674d891104d6da91811dbb89d28ab3faa356ee0ab4cbeae9bba9cf3e971cb"
+  url "https://pecl.php.net/get/apcu_bc-1.0.3.tgz"
```
Comment (javian) [inline Formula/php70-apcu-bc.rb]: Please make this into a separate commit.

**Comment by romainneutron** [comment]
PR updated, comments addressed

**Comment by romainneutron** [comment]
@BrewTestBot test this please

**Comment by romainneutron** [comment]
It seems the build failure is not related to this PR. I'm not sure I've requested a new build correctly. Should I push -f?

**Comment by ilovezfs** [comment]
@BrewTestBot test this please

**Comment by javian** [comment]
@BrewTestBot test this please

**Comment by javian** [comment]
Thank you for the submission @romainneutron !
